outputs.tf

```diff
@@ -2,9 +2,114 @@
 # Outputs
 ##############################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+# Common
+output "region" {
+  description = "Region that instance(s) are provisioned to."
+  value       = var.region
+}
 
+output "resource_group_id" {
```

Comment:
if we already output `logdna_resource_group_id`, `sysdig_resource_group_id` and `activity_tracker_resource_group_id` do we need this output?